pypiserver/__init__.py

```diff
@@ -5,7 +5,7 @@
 import typing as t
 
 from pypiserver.bottle import Bottle
-from pypiserver.config import Config, RunConfig, strtobool
+from pypiserver.config import DEFAULTS, Config, RunConfig, strtobool
```

Comment:
I think we should keep the `DEFAULTS` as much as possible to the `config.py` module, so as to prevent the mixture of responsibilities :) I elaborate on it in other comments.

```suggestion
from pypiserver.config import Config, RunConfig, strtobool
```